packages/framework-provider-azure-infrastructure/src/infrastructure/types/functionDefinition.ts

```diff
@@ -0,0 +1,4 @@
+export interface FunctionDefinition {
+  name: string
+  config: object
```

Comment:
Is there any way to have a more specific type for the config property? Maybe you can import the type from the Azure package or something...

Reply:
Let me see if I'm able to figure out the definition (there is not an azure package for it)